webnode/src/components/storage/toolbox/TreasureTable.tsx

```diff
@@ -1,7 +1,7 @@
-import React from "react";
-import { Table, Header } from "semantic-ui-react";
+import * as React from "react";
```

Comment:
here is another example that can be reverted with the TS compiler option

Reply:
How would you like have it? Would you like to return state and then use without * as  ?